ut-user/index.js

```diff
@@ -16,12 +20,28 @@ function getCrypt(cryptKey) {
     return crypt;
 }
 
+function getIPAddress() {
+  var interfaces = os.networkInterfaces();
+  for (var devName in interfaces) {
+    var iface = interfaces[devName];
+
+    for (var i = 0; i < iface.length; i++) {
+      var alias = iface[i];
+      if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal)
+        return alias.address;
+    }
+  }
+  return '0.0.0.0';
+}
+
 module.exports = {
     init: function(b) {
         getCrypt(b.config.masterCryptKey);
         importMethod = b.importMethod.bind(b);
         checkMethod = b.config['identity.check'];
         debug = b.config.debug;
+        applicationVersion = b.config.applicationVersion;
```

Comment:
My second question is that this requires the above module's module.exports be changed to export a function rather than an object - so we have to think about the implications of it.

On the other hand, if we have a way to get the version of implementation project from the other modules , then that would be a better solution over this.